html-and-web/html-and-the-web_personal-website/index.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Personal Website</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1>Ruslan Mirasov</h1>
+      <nav class="nav">
+        <a href="#contact">Contact</a>

```

Comment:
I'm not sure if the # is necessary. Could you check it!

Reply:
It's fine, this page won't link to other pages, the links should just take you to a specific section on the current page. So it's okay.

Reply:
Well done!

Reply:
ok!